I have no idea why the _run docker build_ stage failed https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=26937 
```
Attempting to finalize metadata for griddataformats
Error: Failed to render jinja template in /home/conda/recipe_root/meta.yaml:
'PYTHON' is undefined
##[error]Bash exited with code '1'.
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -14,7 +14,7 @@ source:
 build:
   number: 1
   noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: "python -m pip install . --no-deps -vv"
 
 requirements:
   build:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,33 +1,27 @@
 {% set name = "GridDataFormats" %}
 {% set version = "0.4.1" %}
-{% set sha256 = "b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3" %}
 
 package:
-  name: {{ name|lower }}
+  name: griddataformats
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/G/GridDataFormats/GridDataFormats-{{ version }}.tar.gz
+  sha256: b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3
 
 build:
   number: 1
   noarch: python
-  script: "pip install . --no-deps -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
-    - python
-    - setuptools
-
   host:
+    - python
     - pip
-
   run:
     - python
     - six
-    - numpy
+    - numpy >=1.0.3
     - scipy
 
 test:
@@ -36,7 +30,7 @@ test:
 
 about:
   home: https://github.com/MDAnalysis/GridDataFormats
-  license: LGPLv3
+  license: LGPL-3.0
   license_file: COPYING.LESSER
   summary: 'Reading and writing of data on regular grids in Python'
   description: |

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -14,7 +14,7 @@ source:
 build:
   number: 1
   noarch: python
-  script: "python -m pip install . --no-deps -vv"
+  script: "pip install . --no-deps -vv"
 
 requirements:
   build:

